packages/core/test/util.js

```diff
@@ -514,7 +499,7 @@ async function prepareScenarioOnDisk({
         : defaultPaths.primary
     filesToWrite.push({
       file: primaryPath,
-      content: jsonStringifySortedPolicy(scenario.config),
+      content: `${jsonStringifySortedPolicy(scenario.config ?? {})}`,
```

Comment:
- [ ] this no longer needs to be wrapped in a string since the type of `jsonStringifySortedPolicy` changed